src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloud.java

```diff
@@ -163,12 +167,30 @@ public String getInstanceUniqueId() {
         return String.valueOf(name.hashCode());
     }
 
+    private ComputeClient createClient() {
+        try {
+            ClientFactory clientFactory = new ClientFactory(Jenkins.get(), new ArrayList<>(),
+                    credentialsId);
+            return clientFactory.compute();
+        } catch (IOException e) {
+            LOGGER.log(Level.WARNING, "Exception why creating GCE client", e);
```

Comment:
This should be level.SEVERE I think, as this plugin cannot function without a working client.